src/query/interpreter.cpp

```diff
@@ -2398,6 +2532,11 @@ Interpreter::PrepareResult Interpreter::Prepare(const std::string &query_string,
       prepared_query = PrepareSettingQuery(std::move(parsed_query), in_explicit_transaction_, &*execution_db_accessor_);
     } else if (utils::Downcast<VersionQuery>(parsed_query.query)) {
       prepared_query = PrepareVersionQuery(std::move(parsed_query), in_explicit_transaction_);
+    } else if (utils::Downcast<TransactionQueueQuery>(parsed_query.query)) {
+      spdlog::info("Number of interpreters in else if: {}", interpreter_context_->interpreters->size());
```

Comment:
I think there is no need for this info here

Reply:
Why would this be beneficial to user? Maybe I don't know something

Reply:
was used for debugging, will be del